app/brave_settings_strings.grdp

```diff
@@ -1517,6 +1517,67 @@ Leo. This action cannot be undone.
+    API Key
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_LABEL_TOOLTIP_INFO" desc="A brief description for the input title for adding a custom model">
+    The name of the model as it will appear in the model selection
+              menu
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_REQUEST_NAME_TOOLTIP_INFO" desc="A brief description for the input title for adding model request name">
+    The name of the model as it should appear in the request to the serving framework, i.e. phi-3 (note that if this name doesn’t match exactly what is expected by the serving framework the integration will not work)
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_SERVER_ENDPOINT_TOOLTIP_INFO" desc="A brief description for the input title for adding server endpoint url">
+    The url where your serving framework is listening for requests. If you are not sure, check the serving framework documentation. E.g. for Ollama, this is always http://localhost:11434/v1/chat/completions
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_API_KEY_TOOLTIP_INFO" desc="A brief description for the input title for adding API key">
+    Some serving frameworks may require authentication credentials, such as an API key or an access token. These will be added to the request header.
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_CLOSE_BUTTON_LABEL" desc="A label for a button that closes configuration UI">
+    Close
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_PROXY_NOTE" desc="A note for users when adding model server info">
+    Brave does not proxy these requests, please read privacy terms of the chosen provider.
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_ENDPOINT_ERROR" desc="An error note for users when url is invalidated">
+    Non-local endpoints should employ the HTTPS protocol for secure data transmission.
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_ADD_NEW_BUTTON_LABEL" desc="A label for a button that opens a panel with input forms for a user to add model">
+    Add new model
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_MODEL_LIST_EMPTY_LABEL" desc="A label an alert when the model list is empty.">
+    You haven't connected a model yet. Begin by clicking the button below.
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_YOUR_MODELS_TITLE" desc="A header title for model list section">
+    Your models
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_YOUR_MODELS_DESC_1" desc="A description explaning to connect models within the browser">
+    Connect your self-hosted or third-party hosted LLM to Leo and use them within the browser.
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_YOUR_MODELS_DESC_2" desc="A description explaning what protocols are accepted">
+    Use any OpenAI Chat Protocol compatible framework such as ollama and oogabooga.
```

Comment:
nit: capitalize "Ollama" and remove "Oogabooga"

Reply:
changed